conda_forge_tick/feedstock_parser.py

```diff
@@ -624,14 +633,13 @@ def load_feedstock(
         The sub_graph, now updated with the feedstock metadata
     """
     in_container = os.environ.get("CF_TICK_IN_CONTAINER", "false") == "true"
-    if use_container is None:
-        use_container = not in_container
 
```

Comment:
Since `use_container` has to `bool` this will never be run.

Reply:
Ah that's a bug. The default value should be `None`. Thanks for catching!

Reply:
Okay, I've adapted the code accordingly!